### PR TITLE
Added Windows Image - Single VM and Multiple VM test

### PR DESCRIPTION
**Pull request by yarunachalam**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -27,3 +27,4 @@ rancher-endpoint: ~
 
 node-scripts-location: 'scripts/vagrant'
 
+win-image-url: 'http://10.84.144.252/meera/ws2016.qcow2'
```
Comment (mbelur) [inline config.yml:30]: can we set the default value of win-image-url to None and in the test check if it's not configured as None, then run the test?

**Review by mbelur — CHANGES_REQUESTED** [review]

File: config.yml (hunk 1 above)
Comment (guangyee) [reply config.yml:30]: We'll need a conditional market for this. I'll implement it in a follow-on patch.

**Review by guangyee — APPROVED** [review]
LGTM

File: config.yml (hunk 1 above)
Comment (mbelur) [reply config.yml:30]: ok sounds good.

**Review by mbelur — APPROVED** [review]

**Pull request merged**